src/models/job.jsx

```diff
@@ -10,6 +10,8 @@ const JobContext = createContext({
   updateJob: () => {},
```

Comment:
for your reference
~~~js
/**
   * job update that does not include change in owner & company
   * @param {} payload: job data with job id
   */
  const updateJob = async (payload) => {
    const jobId = payload.id;
    payload.id = undefined;
    // why use runTransaction? It may happen when you try to update a doc that may be deleted
    await runTransaction(database, async (transaction) => {
      const jobDocRef = doc(database, "Jobs", jobId);
      const jobDoc = await transaction.get(jobDocRef);
      if (!jobDoc.exists()) {
        throw `Job ${jobId} does not exist`;
      }
      transaction.update(jobDocRef, payload);
    });
  };
~~~